modules/admin_manual/pages/enterprise/collaboration/collabora_online_integration.adoc

```diff
@@ -8,34 +8,19 @@ These limitations include:
 * No downloading
 * No editing
 * Watermarking
-+
-NOTE: Administrators can configure the watermark text.
-* Optional printing and exporting to PDF with watermarks included
+* Optional printing and exporting to PDF with watermarks included, which can be adjusted
 
 IMPORTANT: Documents never leave the server when shared with Secure View.
 Collabora Online Server opens them and streams the files to the user's browser (much like a video stream).
 Consequently, there's no way to extract the original document from the browser.
 
-== Supported File Formats
-
-Secure View only supports a limited number of file formats; these are:
-
-* Microsoft Word (.docx)
-* Microsoft Excel (.xlsx)
-* Microsoft PowerPoint (.pptx)
-* OpenDocument Text Document (.odt)
-* OpenDocument Presentation Document (.odp)
-* OpenDocument Spreadsheet Document (.ods)
-* PDF
-
-If a folder shared with Secure View contains unsupported file types (e.g., JPG), they will not be accessible in any way.
-
 == Prerequisites
 
-* ownCloud 10.3 or higher _Enterprise Edition_
-* Collabora Online set up and integrated
+* ownCloud **10.3** or higher _Enterprise Edition_
+* ownCloud Collabora Online App (richdocuments app) Version **2.2.0**
+* Collabora Online Server **4.0.10** set up and integrated
```

Comment:
Do you want these bolded and italicised, or just bolded?

Reply:
just bold

Reply:
Thanks.